site/content/tutorial/01-introduction/05-nested-components/text.md

```diff
@@ -2,17 +2,21 @@
 title: Nested components
 ---
 
-It would be impractical to put your entire app in a single component. Instead, we can import components from other files and include them as though we were including elements.
+It would be impractical to put your entire app in a single component. Instead, we can import components from other files and then use them as though we were including elements.
```

Comment:
```suggestion
It would be impractical to put your entire app in a single component. Instead, we can import components from other files and use them as though we were including elements.
```
Personal preference, but I think this would make it even clearer.